src/core_modules/capture-core/components/LockedSelector/QuickSelector/QuickSelector.component.js

```diff
@@ -119,8 +120,9 @@ class QuickSelector extends Component<Props> {
                         <ActionButtons
                             selectedProgramId={this.props.selectedProgramId}
                             onStartAgainClick={this.props.onStartAgain}
-                            onFindClick={this.props.onClickFind}
-                            onNewClick={this.props.onClickNew}
+                            onFindClick={this.props.onFindClick}
```

Comment:
Changed this bit coz I neglected it on some previous PR. So was inconsistent on this component now.